docs/design.md

```diff
@@ -183,18 +299,24 @@ Not implemented yet.
 
 Steps 3 and 4 can take some time, so the driver must ensure idempotency somehow.
 
-#### NodeUnstageVolume
-
-Just unmount the volume.
-
 #### NodePublishVolume
 
 Just bind-mount the volume.
 
+#### NodeUnstageVolume
+
+Just unmount the volume.
+
 #### NodeUnpublishVolume
 
 Just unmount the volume.
 
+#### NodeExpandVolume
+
+If the attached volume has been formatted with a filesystem, resize the filesystem.
+
+#### NodeGetVolumeStats
+
```

Comment:
Empty section?

Reply:
Forgot to leave a TODO here. 

Might add the following: "Returns the amount of available/total/used bytes and inodes for a given volume.